regression/tests/enterprise/test_enterprise_login.py

```diff
@@ -4,102 +4,118 @@
 import os
 from bok_choy.web_app_test import WebAppTest
 
+from regression.pages import LOGIN_EMAIL, LOGIN_PASSWORD
 from regression.pages.lms.dashboard_lms import DashboardPageExtended
 from regression.pages.lms.login_lms import LmsLogin
-from regression.pages.enterprise.enterprise_logistration_page import (
-    EnterpriseLogistration
+from regression.pages.enterprise.enterprise_edx_logistration_page import (
+    EnterpriseEdxLogistration
+)
+from regression.pages.enterprise.enterprise_portal_login_page import (
+    EnterprisePortalLogin
+)
+from regression.pages.enterprise.enterprise_portal_home_page import (
+    EnterprisePortalHome
+)
+from regression.pages.enterprise.enterprise_portal_course_page import (
+    EnterprisePortalCourseStart,
+    EnterprisePortalCourseStructure
+)
+from regression.pages.enterprise.enterprise_course_enrollment_page import (
+    EnterpriseCourseEnrollment
 )
-from regression.pages.enterprise.idp_page import IDPLogin
 from regression.pages.enterprise.user_account import UserAccountSettings
 from regression.pages.enterprise.enterprise_const import (
     ENTERPRISE_NAME,
     IDP_CSS_ID
 )
-from regression.tests.helpers.api_clients import LmsLoginApi
+from regression.tests.helpers.api_clients import LmsSessionApi
 
 
 class TestEnterpriseLogin(WebAppTest):
     """
     Test Enterprise Login
     """
-
-    EDX_USER_EMAIL = os.environ.get('USER_LOGIN_EMAIL')
-    EDX_USER_PASSWORD = os.environ.get('USER_LOGIN_PASSWORD')
-    IDP_USERNAME = os.environ.get('IDP_USERNAME')
-    IDP_PASSWORD = os.environ.get('IDP_PASSWORD')
+    ENTERPRISE_PORTAL_USERNAME = os.environ.get('ENTERPRISE_PORTAL_USERNAME')
```

Comment:
I am using TestCourseDataSAP and mtd_learner7, however this information is stored in environment variables so there is no dependency and we can use any user/course on run time